admin/bin/qserv-admin.py

```diff
@@ -273,6 +274,12 @@ def _parseCreateNode(self, tokens):
         # check that all required options are there
         self._checkExist(options, requiredKeys)
 
+        # check obsolete keys
+        for opt in ('runDir', 'mysqlConn'):
+            if opt in options:
+                self._logger.warning(opt + ' option is obsolete')
+                del options[opt]
```

Comment:
This is temporary thing to allow migration for the existing clients, after some time it will generate exception for obsolete/unknown options